book/content/credit/credit.md

```diff
@@ -79,6 +83,50 @@ It's perfectly possible to cite a dataset or software package directly, and most
+
+See also [this example](https://www.force11.org/node/4771) provided by FORCE11.
+
+### Where to cite data in papers?
+
+You should cite your dataset directly in the paper in places where it is relevant, just like publications, as well as in a **data availability statement** at the end of the paper (similar to the acknowledgement section). You can find examples of these statements in the publishers' (research data) author policies, or below:
+    
+### Data availability statement examples:
+**Using the Digital Object Identifier (DOI):**
+“The data that support the findings of this study are openly available in [repository name] at http://doi.org/[doi].”
+
+**If no DOI is issued:** 
+- “The data that support the findings of this study are openly available in [repository name] at [URL], reference number [reference number].”
+
+**When there is an embargo period you can reserve your DOI and still include a reference to the dataset in your paper:**
+- “The data that support the findings will be available in [repository name] at [URL / DOI] following a [6 month] embargo from the date of publication to allow for commercialisation of research findings.” 
+
+**When data cannot be made available:**
+- “Restrictions apply to the data that support the findings of this study. [Explain nature of restrictions e.g. the data contains information that could compromise the privacy of research participants] Data are available upon reasonable request by contacting [name and contact details] and with permission of [third party name].” 
+-  “The data that support the findings of this study are available upon request. Access conditions and procedures can be found at [URL to restricted access repository such as [EASY](https://easy.dans.knaw.nl/ui/home).]”
+
+## How to make software citeable?
+
+A software citation a lot of the same elements as a data citation, described above. To make your code citeable, you can use the integration between [Zenodo](https://zenodo.org/) and GitHub 
+ 
+- Create a file to tell people how to cite your software. Use this [handy guide](https://citation-file-format.github.io/cff-initializer-javascript/) to format the file. 
+- Link your GitHub account with a Zenodo account. This guide explains [how](https://guides.github.com/activities/citable-code/).  
+- You can tell Zenodo what information or metadata you want to include with your software by adding a `zenodo.json` file, described [here](https://guide.esciencecenter.nl/citable_software/making_software_citable.html). 
+- On Zenodo, flip the swith to the 'on' position for the GitHub repository you want to release
+- On GitHub, click the *Create a new release* button. Zenodo should automatically be notified and should make a snapshot copy of the current state of your repository (just one branch, without any history), and should also assign a persistent identifier (DOI) to that snapshot.
+- Use the DOI in any citations of your software!
```

Comment:
It is also about others citing your software, not necessarily you citing your software.